solutions/src/main/java/ru/sberbank/school/task02/FxConverter.java

```diff
@@ -1,43 +1,48 @@
 package ru.sberbank.school.task02;
 
 import java.math.BigDecimal;
-import java.util.List;
 
+import lombok.NonNull;
+import ru.sberbank.school.task02.exception.FxConversionException;
 import ru.sberbank.school.task02.util.ClientOperation;
 import ru.sberbank.school.task02.util.Quote;
 import ru.sberbank.school.task02.util.Symbol;
-import ru.sberbank.school.task02.util.Volume;
 
 public class FxConverter implements FxConversionService {
 
-    private ExternalQuotesService Quotes;
+    private ExternalQuotesService quotesService;
 
     FxConverter(ExternalQuotesService q) {
-        Quotes = q;
+        quotesService = q;
     }
 
-    @Override
-    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) {
-        if (operation == null) {
-            throw new NullPointerException("Operation is null");
-        }
 
-        if (symbol == null) {
-            throw new NullPointerException("Symbol is null");
-        }
+    /**
```

Comment:
First sentence of Javadoc is missing an ending period.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1549488121>*